[rocBLAS][Tensile] Add initial gfx1152/gfx1153 support

amd-mtrifuno opened the pull request.

amd-mtrifuno committed:
Add initial Tensile gfx1152 support

amd-mtrifuno committed:
Add initial rocBLAS gfx1152 support

amd-mtrifuno committed:
Rename yaml files

TorreZuk commented on projects/rocblas/CMakeLists.txt:
```diff
@@ -82,7 +82,7 @@ if (NOT BUILD_ADDRESS_SANITIZER)
   set( TARGET_LIST_ROCM_6.3 "gfx900;gfx906:xnack-;gfx908:xnack-;gfx90a;gfx942;gfx1010;gfx1012;gfx1030;gfx1100;gfx1101;gfx1102;gfx1151;gfx1200;gfx1201")
   set( TARGET_LIST_ROCM_7.0 "gfx900;gfx906:xnack-;gfx908:xnack-;gfx90a;gfx942;gfx950;gfx1010;gfx1012;gfx1030;gfx1100;gfx1101;gfx1102;gfx1150;gfx1151;gfx1200;gfx1201")
-  set( TARGET_LIST_ROCM_7.1 "gfx900;gfx906:xnack-;gfx908:xnack-;gfx90a;gfx942;gfx950;gfx1010;gfx1012;gfx1030;gfx1100;gfx1101;gfx1102;gfx1103;gfx1150;gfx1151;gfx1200;gfx1201")
+  set( TARGET_LIST_ROCM_7.1 "gfx900;gfx906:xnack-;gfx908:xnack-;gfx90a;gfx942;gfx950;gfx1010;gfx1012;gfx1030;gfx1100;gfx1101;gfx1102;gfx1103;gfx1150;gfx1151;gfx1152;gfx1200;gfx1201")
```
rocBLAS changes all good,  but this line we won't usually change list for old release unless a good reason.  @amcamd

amd-mtrifuno replied:
I will delete gfx1152 from the 7.1 target list, we only need these changes on the develop branch.

amd-mtrifuno committed:
Remove gfx1152 from TARGET_LIST_ROCM_7.1

TorreZuk reviewed (approved):
rocblas side looks good

mgehre-amd committed:
[rocblas][Tensile] Initial support for gfx1153

Tensile/Logic/asm_full/gfx1153/*.yaml files are copied from gfx1152 with the following changes:
- replaced gfx1152 in filename and content by gfx1153
- changed device to `[Device 1902]` (PCI id)
- changed isa to `ISA: [11, 5, 3]`

TorreZuk commented:
Other targets were merged so @amd-mtrifuno please resolve conflicts and re-run CI,  I can help push for a review after that.   thanks.

amd-mtrifuno committed:
Merge branch 'develop' into users/mtrifuno/rocblas_tensile_add_gfx1152

amd-mtrifuno commented:
> Other targets were merged so @amd-mtrifuno please resolve conflicts and re-run CI, I can help push for a review after that. thanks.

Done

TorreZuk commented:
@bstefanuk if the failures match the baseline and yaml follow your new scheme can you merge this as gardener?  Or should we trigger re-run?

mgehre-amd committed:
Merge branch 'develop' into users/mtrifuno/rocblas_tensile_add_gfx1152

mgehre-amd commented:
[Failed again](https://math-ci.amd.com/job/rocm-libraries/job/precheckin/job/rocblas/job/PR-2653/8/pipeline-overview/) with
```
[----------] 4354 tests from _/tbmv
corrupted double-linked list
```
on gfx950 which seems unrelated. How can we get this PR merged?

TorreZuk commented:
Sorry I don't have permissions to help, you need a gardener.  Yes that failure is in baseline so not related to your change.
